README.md

```diff
@@ -18,7 +18,7 @@ Our goal will be to make our page look like this:
 
 ![demo app](https://curriculum-content.s3.amazonaws.com/phase-2/phase-2-hooks-jsx-lab/demo.png)
 
-Once your app matches the demo image, run `learn test` or `npm test` to check
+Once your app matches the demo image, run `npm test` to check
 your work.
 
 ## Deliverables
```

Comment:
On line 39, change to: "... there are a couple of variables being imported that will..."

On line 51: missing "for", "... React documentation **for** the `style` property..."